src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -128,6 +129,13 @@ public Object instanceMainPostParseArgs() {
             System.setProperty("java.io.tmpdir", f.getAbsolutePath()); // in loop so that last one takes effect
         }
 
+        //Set defaults (note: setting them here means they are not controllable by the user)
+        BlockCompressedOutputStream.setDefaultDeflaterFactory(new IntelDeflaterFactory());
```

Comment:
Add a command-line arg to `CommandLineProgram` that allows the user to opt-out of using the `IntelDeflater`